unblob/processing.py

```diff
@@ -42,6 +36,8 @@ class ExtractionConfig:
     max_depth: int = DEFAULT_DEPTH
     process_num: int = DEFAULT_PROCESS_NUM
     keep_extracted_chunks: bool = False
+    eliminate_whole_file_chunks: bool = True
+    extract_suffix: str = "_extract"
```

Comment:
do we want to make this configurable?

Reply:
not necessarily, but the constant for suffix does not really hurt either at this location.

I could also think of it being configurable - in case of file name collision during extraction, which we do not handle at the moment.